pkg/controller/extensions/request/extensionrequest_test.go

```diff
@@ -817,6 +834,7 @@ func TestDiscoverExecutorInfo(t *testing.T) {
 
 			os.Setenv(util.PodNameEnvVar, "podName")
 			os.Setenv(util.PodNamespaceEnvVar, "podNamespace")
+			os.Setenv(PodImageNameEnvVar, tt.imageName)
```

Comment:
Would it make sense to put `PodImageNameEnvVar` in `util` (like the other env var names)?

Reply:
I considered that but went with `request.PodImageNameEnvVar` to avoid polluting what seemed to me to be [core-crossplane space (pkg/util)](https://github.com/crossplaneio/crossplane/blob/6a9ede4511b4649b8cd7ceb9ba48a4a11f4da800/pkg/util/pod.go#L31-L36) with an extension-manager specific constant.

I could go either way.  If we split things up later we can deal with it then.

Reply:
Awesome. Would it make sense to add that extra context to the comments for the constant, so future readers can get the context easily?

Reply:
I'm making a few other changes to this variable that might distinguish this variable from the others. Let me know if some supporting comments would still be helpful.